middleware.go

```diff
@@ -12,7 +12,7 @@ type middleware interface {
 	Middleware(handler http.Handler) http.Handler
 }
 
-// MiddlewareFunc also implements the middleware interface.
+// Middleware also implements the middleware interface.
```

Comment:
Maybe it'd be a good idea to remove the word "also":

```
// Middleware implements the middleware interface.
```

When reading documentation for this method, it might be confusing. "Middleware _also_ implements it? In addition to what?"